tools/cldr-code/src/main/java/com/ibm/icu/dev/test/TestFmwk.java

```diff
@@ -27,13 +27,18 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.logging.Logger;
```

Comment:
Are you planning to sync the changes here back into https://github.com/unicode-org/icu/blob/main/icu4j/main/tests/framework/src/com/ibm/icu/dev/test/TestFmwk.java ?

That would be good...

And what about the equivalent feature in ICU4C tests?

Reply:
Is it still used in ICU4J? I thought those were ported over.

Reply:
> Is it still used in ICU4J? I thought those were ported over to junit.

Yes, TestFmwk.java is still there, and is used. The ICU version *calls* JUnit.

Reply:
hm. TestFmwk.java in ICU is _quite_ divergent.  CLDR picked up 57.1, probably pre JUnit.